layout/_partials/post/post-followme.swig

```diff
@@ -0,0 +1,23 @@
+{%- if theme.follow_me.enable %}
+  {%- set builtin = ['twitter', 'weibo', 'telegram', 'wechat'] %}
+
+  <div class="followme flex-column align-center">
+    <p>欢迎关注我的其它发布渠道</p>
+
+    <div class="flex align-center social-list">
+
+        {%- for name, link in theme.follow_me.channels %}
+          {%- if builtin.includes(name) %}
+            {%- set icon_url = __('/images/follow-me/' + name + '.svg') %}
```

Comment:
How about using Font Awesome icons here?

I'm not sure if this is a good idea, as the font awesome icon has no color, but does not require additional svg files

Reply:
Good idea. let me try this with optional color customization.

Reply:
🤔made another version by Font Awesome with latest commit, any suggestion? @stevenjoezhang 
![image](https://user-images.githubusercontent.com/6239652/73326209-62489500-428c-11ea-970c-a6d5d29def69.png)

Reply:
It looks great! I will do some tests before merging this Pull Request